### ISSUE TITLE
data loss when reading from a sav file

### ISSUE THREAD

**Issue by spsspro** [Roche/pyreadstat#188]
Please read the README, particularly the known limitations section!

Describe the issue
> missing column when reading from a sav file

To Reproduce
```
from pyreadstat import read_sav
df, meta = read_sav('b9d238e8432342ea9204e3d3ae60ca44.sav')
```
> the origin sav has 176 column, but the df only 173 column

File example
> sample file can download from here: https://easyupload.io/t70uu8

Expected behavior

Setup Information:
How did you install pyreadstat? (with pip)
Platform? (M1 macOS, linux)
Python Version? (Python3.9)
Python Distribution? (System)
Using Virtualenv or condaenv? ( virtualenv)

**Comment by ofajardo** [comment]
Thanks for the reproducible report. 

The problem is that there are 3 columns that are duplicated. 

The issue now is that I am using dictionaries to store the data, and more recently also returning dictionaries on demand so that data can be imported into polaris more efficiently, so I cannot allow to have duplicated column names. 

Interestingly neither SPSS or PSPP allow duplicated column names. If you open a file with duplicated column names, it renames one of the columns automatically. You also cannot manually assign an existing name to a new column. So this tells me that in general this should not be allowed. 

How did you generate the file? If it was using pyreadstat, then what I have to do is to correct the writing function to raise an exception if duplicated column names. If you used something else to generate the file, then I guess I could rename the duplicated columns (and in addition correct the writing function anyway)

**Comment by ofajardo** [comment]
OK changes are in dev: 
* An error is raised if the user attempts to write a file with duplicated columns
* If duplicated columns are detected when reading, a warning is raised and the column is renamed to *_duplicated1 where 1 will increase if more duplicates are detected

**Comment by spsspro** [comment]
> OK changes are in dev:
> 
> * An error is raised if the user attempts to write a file with duplicated columns
> * If duplicated columns are detected when reading, a warning is raised and the column is renamed to *_duplicated1 where 1 will increase if more duplicates are detected

**Comment by ofajardo** [comment]
solved in 1.1.7
